packages/tadviewer/src/components/DataGrid.tsx

```diff
@@ -638,7 +638,8 @@ export interface DataGridProps {
     column: number,
     dataRow: DataRow,
     columnId: string,
-    cellVal: any
+    cellVal: any,
+    columnName: string,
```

Comment:
I don't think we need to pass `columnName` here since we already have `columnId` passed as an argument to the callback.

Reply:
Do we know that the `columnId` will always match the column name? I am not able to navigate to see types easily - if that's the case that'd be great

Reply:
Yep!  Up above you are passing `col.field` as the column name to `onGridClick`, where `col` is the [SlickGrid column object](https://github.com/6pac/SlickGrid/wiki/Column-Options).  This object gets created by `mkSlickColMap()`, which always uses the column id for both the `id` and `field` fields.

Reply:
Fantastic, thanks for that link. Updated